testing/helpers/includeThemesLinks.js

```diff
@@ -0,0 +1,9 @@
+const themesList = ['generic.light', 'material.blue.light'];
+
+themesList.forEach(theme => {
+    const link = document.createElement('link');
+    link.setAttribute('rel', 'dx-theme');
+    link.setAttribute('data-theme', theme);
+    link.setAttribute('href', SystemJS.normalizeSync(theme.replace(/\./g, '_') + '.css'));
+    document.head.appendChild(link);
+});
```

Comment:
Can't we use `window.includeThemesLinks();` here to prevent code duplication?

Reply:
Is it a problem to make this change for that test too? I think it's better to do it right away.

Reply:
We can update that test in this PR.